guide/contribute/propose.md

```diff
@@ -0,0 +1,149 @@
+- Browse [bitcoin.design](https://bitcoin.design/guide) test it and try to identify issues yourself
+
+For more challenging pieces of work, a new page, or a new chapter, it's a good idea to talk to the community and seek consensus before you begin working on it. This helps avoid work duplication and makes sure your efforts are aligned with the community-determined goals.
+
+### Getting started
+
+Like any novelty, the process may seem intimidating at first, but the best way to understand this is to try it out in practice.
+
+{% include youtube.html id="DxrLZtABG0E" %}
+
+1. [Create a fork (clone)](#forking-design-guide-repository)
+2. [Create a branch](#creating-a-branch)
+3. [Manage files](#managing-files)
+4. [Apply changes](#applying-changes)
+5. [Create a pull request](#creating-a-pull-request)
+6. [Applying additional changes](#applying-additional-changes)
+
+#### Forking Design Guide repository
+
+The first step is to fork the Bitcoin Design Guide repository. Forking a repository means that you're creating an exact clone (copy) of the Bitcoin Design Guide locally on your computer and in a cloud on your GitHub.com account.
+
+To fork a repository[^1]:
+
+- Sign in to GitHub desktop
+- File > `Clone repository`
+- Switch to the URL tab
+- Paste `https://github.com/BitcoinDesign/Guide/` into the URL field.
+- Click `Clone`
+
+{% include picture.html
+  image = "/assets/images/guide/contribute/propose/clonerepo.png"
+%}
+
+#### Creating a branch
+
+A branch[^2] allows you to create a copy of a copy. This sets up a _contained area_ where you can apply changes, without going out of sync with the cloned version.
+
+The cloned version should reside on a `master/main branch`, whereas any changes should be done on separate branches. This isn't a strict rule, it's the best practice that would allow you to easier sync your copy with the original (upstream) version of the design guide.
+
+{% include picture.html
+  image = "/assets/images/guide/contribute/propose/createbranch.png"
+%}
+
+#### Managing files
+
+When a new branch is created, while making sure it's selected, click on either `Open in Text Editor` or`Show in explorer` to browse through the folders and files. You can use a search (find) function `Ctrl+F/ cmd+F` to search for a specific keyword that can help identify the file you would like to edit.
+
+#### Applying changes
+
+After identifying a file or folder where you want to make a change, it's time to make your changes to the text, code, or visuals. Whenever you're finished, make sure to `Save` the file (File > Save) in a _text editor_ or use a shortcut `Ctrl+S/cmd+S`.
+
+Please make sure that your changes are consistent with our [content](content-guidelines), [illustration](illustration-guidelines) and [formatting](formatting) guidelines.
+
+{% include youtube.html id="jlt5GDko-to" %}
+
+##### Committing
```

Comment:
anything going to be stated regarding compression, previewing the change, and the directory for images?

Reply:
I'm not sure. Compression is something that can change in the future?

What do you mean by `directory for images`?

Building locally and previewing changes is covered in the video. Do you think I need to cover it as a special paragraph? I didn't want to duplicate as I want to refer to that in the review.md where I will tackle the review process.